### ISSUE TITLE
needs Cache-Control headers with max-age

### ISSUE THREAD

**Issue by reedstrm** [openstax/exercises#183]
In order to effectively cache the results of exercise look ups, it would be good if the server, in addition to providing ETags and supporting If-None-Match (which it does), would provide a cache-control header with a reasonable value, fairly large, since exercises do not change often.

**Comment by reedstrm** [comment]
current value is 'max-age=0, private, must-revalidate', which I think actually saves 0 time.  In my measurements of If-None-Match vs. full fetches of exercises, I get the same total fetch time.

**Comment by Dantemss** [comment]
Yeah ETags would only save the data transmission time which is tiny with a fast internet connection in comparison to the time it takes to generate the JSON.

I created a change request ticket to improve the cache headers and cache the result JSON somewhere: https://trello.com/c/ZNuzEBDK

**Comment by nathanstitt** [comment]
this is good but I think it'll only help us in the rare case of a web-browser download.

For instance I'm not seeing how the the Tutor importer could supply a meaningful only-if-modified-since header.

**Comment by Dantemss** [comment]
ETags already work :) Only Cache-Control still needs to be added. I think for published exercises we can set it to 1 year.
